### PR TITLE
composer: allow Symfony 3.x

### PR DESCRIPTION
**Pull request by MacDada**
#8

### CODE REVIEW

**Comment by dantleech** [comment]
hey, the build is failing because PHP 5.5 doesn't seem to be available on travis anymore, would be great if you could update, otherwise I can have a look when I have 10 minutes.

**Comment by MacDada** [comment]
> hey, the build is failing because PHP 5.5 doesn't seem to be available on travis anymore, would be great if you could update (maybe just use PHP 7.0), otherwise I can have a look when I have 10 minutes.

done

**Comment by dantleech** [comment]
Thanks for that - would ultimately be good to add support for 4.x and 5.x too and update Travis accordingly, but I guess we can do that if and when it's required ...

**Comment by dantleech** [comment]
Also should probably drop 5.3 support and start support at f.e. `7.1`

File: tests/ArrayToDelimitedStringTransformerTest.php [hunk 1]
```diff
@@ -114,7 +116,8 @@ public function provideTransform()
     public function testTransform($array, $string, $delimiter = null, $paddingLeft = null, $paddingRight = null, $expectedException = null)
     {
         if ($expectedException) {
-            $this->setExpectedException('Symfony\Component\Form\Exception\TransformationFailedException', $expectedException);
+            $this->expectException('Symfony\Component\Form\Exception\TransformationFailedException');
```
[the remark this answers is not in the document]
Comment (MacDada) [reply tests/ArrayToDelimitedStringTransformerTest.php:119]: > Could replace with `TransformationFailedException::class`

Yeah, I didn't know if you want to bump php version requirement for the lib.

**Comment by MacDada** [comment]
> Thanks for that - would ultimately be good to add support for 4.x and 5.x too and update Travis accordingly, but I guess we can do that if and when it's required ...

I might need that next week, haha

**Pull request merged**